### ISSUE TITLE
Waiting for League to start...

### ISSUE THREAD

**Issue by AndyKXu** [Pupix/rift-explorer#21]
So I have looked at the posts of the same issue and I have tried running the program as administrator, but it still does not get past the: "Waiting for League to start..."

Both League Client and the program are being ran as administrators and I have the enable_swagger: true in the system.yaml file.

My League Client is on my D: drive while the program is on the C: drive.

I am on the NA server

**Comment by Pupix** [comment]
That's quite strange.
Can you run `WMIC PROCESS WHERE name='LeagueClientUx.exe' GET commandline` in a console and post the results here?

**Comment by AndyKXu** [comment]
C:\Users\troll>SET PATH=C:\Windows\System32\wbem\;%PATH%;

C:\Users\troll>WMIC PROCESS WHERE name='LeagueClientUx.exe' GET commandline
CommandLine

Is what I get

**Comment by Pupix** [comment]
Was the client open when you did that? It needs to be open.

If it was already open then the executable must have a different name. The list of processes would also help.

**Comment by AndyKXu** [comment]
Yes it has already opened,
[As you can see here, the executable is shown in the list](https://i.imgur.com/gZU1j2u.png)

**Comment by Pupix** [comment]
That's quite strange. When this happened on other regions it was because the client was started with admin  permissions and the console or Rift Explorer were not. 🤔

**Comment by AndyKXu** [comment]
Nope, the only flag is the swagger. 

It is strange, but I'll give it a try when I'm back on my main computer in a few weeks

**Comment by bstadick** [comment]
I was having a similar issue as well. I have the game installed at a non-default path "C:\Program Files (x86)\Riot Games". I found that when I cloned rift-explorer and did yarn install it didn't get the latest version [v1.0.2](https://github.com/Pupix/lcu-connector/releases/tag/v1.0.2) of lcu-connector (took 1.0.1) so it didn't have the [chokidar fix](https://github.com/paulmillr/chokidar/issues/300) for handling special glob characters (such as the parenthesis). I had to update the package.json of rift-explorer to explicitly take "lcu-connector": "^1.0.2" and re-install. The pre-built may be suffering the same issue.
